docs/source/trainer.rst

```diff
@@ -1278,19 +1278,20 @@ reload_dataloaders_every_epoch
 
 |
 
-Set to True to reload dataloaders every epoch.
+Set to a postive integer to reload dataloaders every n epochs.
 
 .. code-block:: python
 
-    # if False (default)
+    # if 0 (default)
```

Comment:
0 should not be possible.

Reply:
If  `reload_dataloaders_every_n_epochs` should not be 0 , should Trainer allow bool and int so that  `reload_dataloaders_every_n_epochs` can be set to False instead of 0?

Reply:
Ignore this comment.